assets/components/directDebit/directDebitForm/directDebitForm.scss

```diff
@@ -1,79 +1,184 @@
 .component-direct-debit-form {
-  width: 500px;
-  padding: $gu-h-spacing/2;
+  clear: left;
+  margin-top: 80px;
+  margin-left: 0px;
```

Comment:
Is it necessary this `margin-left` property here? The components should not have CSS properties related to the position. If it is necessary, the way of putting 0 is without a unit like:   `margin-left:0`